app/views/plannings/_edit.html.haml

```diff
@@ -248,6 +248,7 @@
   large_plannning = @planning.routes.select{ |route| !route.hidden || !route.locked || route.vehicle_usage_id.nil? }.any?{ |r| (stops_count += r.stops_size) >= 1000 }
 
   controller.js(
+    columns_preferences: current_user.exportable_columns,
```

Comment:
```suggestion
    export_settings: current_user.export_settings,
```
It is no more limited to the columns, so I would prefer to have a more generic naming